angular/src/lib/tabs/examples/default.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'example-tabs-default',
+  template: `
+    <cui-tabs [focus]='0' className='MyClass' (whenSelect)='onClick($event)'>
+        <cui-tab-header heading='test'></cui-tab-header>
+        <cui-tab-list>
+            <cui-tab>Tab A</cui-tab>
+            <cui-tab>Tab B</cui-tab>
+        </cui-tab-list>
+        <cui-tab-content>
+            <cui-tab-pane>Pane A</cui-tab-pane>
+            <cui-tab-pane>Pane B</cui-tab-pane>
+        </cui-tab-content>
+    </cui-tabs>
+  `,
+})
+export class ExampleTabsDefaultComponent {
+
+  dataModel: string = 'Test';
```

Comment:
Doesn't seem to be used

Reply:
> Works well with keyboard, but with voiceover, it has two issues
> 
> 1. It says both the tabs are selected
> 2. It says 1 of 1, instead of 1 of 2
> 
> <img alt="screen shot 2019-03-08 at 9 28 17 am" width="1440" src="https://user-images.githubusercontent.com/2327532/54037876-cf3ed200-4184-11e9-857d-e159f3a5e8ed.png">
> 
> <img alt="screen shot 2019-03-08 at 9 28 21 am" width="1440" src="https://user-images.githubusercontent.com/2327532/54037888-d4038600-4184-11e9-8a0f-6bc8ccadb978.png">
> 
> Angular Material does it properly https://material.angular.io/components/tabs/overview

Find the solution 
add role="presentation" to the container div.
Tabs React component should have the same problem @pauljeter